src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countyCourtJudgment/RepaymentPlanContentProvider.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff.content.countyCourtJudgment;
+
+import uk.gov.hmcts.cmc.claimstore.models.CountyCourtJudgment;
+import uk.gov.hmcts.cmc.claimstore.models.ccj.PaymentOption;
+import uk.gov.hmcts.cmc.claimstore.models.ccj.RepaymentPlan;
+
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatDate;
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatMoney;
+
+public final class RepaymentPlanContentProvider {
+
+    private RepaymentPlanContentProvider() {
+        // Utils class no constructing
+    }
+
+    public static String create(CountyCourtJudgment countyCourtJudgment) {
+        switch (countyCourtJudgment.getPaymentOption()) {
+            case IMMEDIATELY:
+                return PaymentOption.IMMEDIATELY.getDescription();
+            case INSTALMENTS:
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this switch case number of lines from 9 to at most 5, for example by extracting code into methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1151)